airflow/contrib/operators/dataproc_operator.py

```diff
@@ -674,17 +675,17 @@ class DataProcJobBaseOperator(BaseOperator):
 
     @apply_defaults
     def __init__(self,
-                 job_name='{{task.task_id}}_{{ds_nodash}}',
-                 cluster_name="cluster-1",
-                 dataproc_properties=None,
-                 dataproc_jars=None,
-                 gcp_conn_id='google_cloud_default',
-                 delegate_to=None,
-                 labels=None,
-                 region='global',
-                 job_error_states=None,
+                 job_name: str = '{{task.task_id}}_{{ds_nodash}}',
+                 cluster_name: str = "cluster-1",
+                 dataproc_properties: Optional[Dict] = None,
+                 dataproc_jars: Optional[List] = None,
```

Comment:
```suggestion
                 dataproc_jars: Optional[List[str]] = None,
```

Reply:
It would be worth to update types in docs strings.

Reply:
Fixed